src/closestPair.js

```diff
@@ -0,0 +1,36 @@
+const closestPair = array => {
+  if ( array.length < 2 ) {
+    return Infinity
+  } else {
+    let minimumDistance = distance( array[0], array[1] ),
+        minimumPoints = array.slice( 0, 2 ),
```

Comment:
Not a fan of this indentation, it introduces a maintenance requirement to support formatting.  I would consider declaring separately, or indenting with the standard 2 space indentation.